packages/components/src/Badge/Badge.scss

```diff
@@ -1,121 +1,244 @@
+				min-height: $tc-badge-large-height - 2 * $tc-badge-large-vertical-padding;
+
+				svg {
+					height: $tc-badge-large-delete-icon-size;
+					width: $tc-badge-large-delete-icon-size;
+				}
+			}
 		}
 	}
 
-	&-selected:not(&-disabled) {
-		svg,
-		.tc-badge-label,
-		.tc-badge-category {
-			color: $white;
+	&.tc-badge-display-small {
+		.tc-badge-separator {
+			margin-top: 0.3rem;
 		}
 
 		.tc-badge-button {
-			background-color: $scooter;
+			height: $tc-badge-small-height;
+			margin: $tc-badge-small-margin;
+			padding-top: 0;
 
 			.tc-badge-category {
-				background-color: rgba($white, 0.2);
+				font-size: $tc-badge-small-label-font-size;
+				font-weight: $tc-badge-small-label-font-weight;
+			}
+
+			.tc-badge-label {
+				&-text {
+					font-size: $tc-badge-small-label-font-size;
+					font-weight: $tc-badge-small-label-font-weight;
+				}
+
+				&-text-with-categ {
+					font-size: $tc-badge-small-label-font-size;
+					font-weight: $tc-badge-small-label-with-categ-font-weight;
+				}
+
+				&-icon {
+					margin-top: $padding-smaller;
+					height: $tc-badge-small-icon-size;
+					width: $tc-badge-small-icon-size;
+					min-width: $tc-badge-small-icon-size;
+				}
 			}
 
-			&:hover {
-				background-color: darken($scooter, 10);
+			.tc-badge-delete-icon {
+				height: $tc-badge-small-height - 2 * $tc-badge-small-vertical-padding;
+				min-height: $tc-badge-small-height - 2 * $tc-badge-small-vertical-padding;
+
+				svg {
+					height: $tc-badge-small-delete-icon-size;
+					width: $tc-badge-small-delete-icon-size;
+				}
 			}
 		}
 	}
 
-	&-disabled {
+	&-aslink {
 		.tc-badge-button {
-			background-color: $gallery;
+			border-color: rgba($lochmara, 0.25);
 
-			svg,
-			.tc-badge-label,
 			.tc-badge-category {
-				color: $silver;
+				display: none;
 			}
 
-			.tc-badge-category {
-				background-color: $concrete;
+			.tc-badge-separator {
+				display: none;
+			}
+
+			.tc-badge-label {
+				.tc-badge-label-text,
+				.tc-badge-label-icon {
+					color: $lochmara;
+				}
 			}
+		}
+		&:hover:not(.tc-badge-disabled) {
+			.tc-badge-button {
+				border-color: rgba($lochmara, 1);
+			}
+		}
+	}
+
+	&-edit:not(&-disabled) {
+		.tc-badge-label:hover {
+			.tc-badge-label-text,
+			.tc-badge-label-text-with-categ,
+			.tc-badge-label-icon {
+				color: $lochmara;
+			}
+		}
+	}
 
-			& + span {
-				display: flex;
+	&-selected:not(&-disabled) {
+		.tc-badge-label {
+			.tc-badge-label-text,
+			.tc-badge-label-text-with-categ,
+			.tc-badge-label-icon {
+				color: $lochmara;
 			}
 		}
 	}
 
-	.tc-badge-delete-icon {
-		position: relative;
-		margin: auto;
-		height: $tc-badge-height - 2 * $tc-badge-vertical-padding;
-		padding: 0;
-		background: transparent;
-		box-shadow: none;
-		left: -$padding-large;
-
-		svg {
-			display: block;
-			margin: 0;
-			height: $tc-badge-delete-icon-size;
-			width: $tc-badge-delete-icon-size;
+	&-disabled {
+		.tc-badge-button {
+			opacity: 0.62;
```

Comment:
0.62 opacity ? seems very specific 😅
Is it wanted ? If yes, maybe we could put this in a variable top of the file ? 😇